sentry/src/main/java/io/sentry/Session.java

```diff
@@ -73,7 +73,7 @@ public Session(
       final @Nullable String ipAddress,
       final @Nullable String userAgent,
       final @Nullable String environment,
-      final @NotNull String release) {
+      final @Nullable String release) {
```

Comment:
A session without a release is invalid. Why does need to be Nullable?

Reply:
`SentryOptions#release` is nullable. If the release cannot be null on sessions, we need to handle it somehow then. Perhaps @marandaneto has ideas?

Reply:
there's already checks in the code to not create a session if release isn't available, where is it breaking it notnull?

Reply:
`Hub#startSession` -> `Scope#startSession` - startSession uses `SentryOptions#release` which is a nullable field. Technically its possible that null sneaks in into a session. We can either disable the null check for this method or make `Scope#startSession` return `null` or throw if the `release` is `null`.

Reply:
Can we return null in that case?

Reply:
Changed to return `null`.